assets/src/components/template-inserter/index.js

```diff
@@ -105,18 +105,23 @@ class TemplateInserter extends Component {
 										/>
 									</div>
 									{ storyTemplates && storyTemplates.map( ( item ) => (
-										<Button
+										<a // eslint-disable-line jsx-a11y/anchor-is-valid, see https://github.com/ampproject/amp-wp/issues/2165
 											key={ `template-preview-${ item.id }` }
+											role="button"
+											tabIndex="0"
 											onClick={ () => {
 												onSelect( item );
 											} }
+											onKeyDown={ () => {
+												onSelect( item );
```

Comment:
Haven't tested, but this should probably check for what kind of key was pressed. I guess `ENTER` and `SPACE` would be the only ones. `@wordpress/keycodes` can help with that.

Reply:
Ah, good point. 

https://github.com/ampproject/amp-wp/pull/2206/commits/45c7246e79303edf90060be33bf658496a7cdeee checks that it's an `ENTER` or `SPACE` key.